File: Pascal/Task255.pas
program Task255;
var a, b, c, d, u, x, hx, y, hy : double;
    i, j : integer;
begin
    writeln('Задайте відрізок для змінної x');
    readln( a, b);{з консолі можна читати кілька значень}
    x := a; hx := (b - a) / 7;
    writeln('Задайте відрізок для змінної y');
    readln( c, d);
    y := c; hy := (d - c) / 7;
    write(' y\x ');{подальший вивід буде в тому ж рядку}
    {виводимо заголовок таблиці зі значеннями х}
    for i := 0 to 7 do write((x + hx * i):8:3);
    writeln();{перехід на новий рядок}
    {Зовнішній цикл за аргументом y}
    for i := 1 to 8 do
    begin
        write(y:5:2);
        x := a; {вкладений цикл за аргументом х}
        for j := 1 to 8 do
        begin
            u := sqrt(2 * x * y + 4 * y * y * y);
            write(u:8:2);
            x := x + hx;
        end;
        writeln();
        y := y + hy;
    end
end.

File: C#/Task255.cs
using System;

public class Task255
{
    public static void Main()
    {
        Console.WriteLine("Введіть межі для x");
        double a = double.Parse(Console.ReadLine());
        double b = double.Parse(Console.ReadLine());

        double x = a;
        double hx = (b - a) / 7.0;

        Console.WriteLine("Введіть межі для y");
        double c = double.Parse(Console.ReadLine());
        double d = double.Parse(Console.ReadLine());

        double y = c;
        double hy = (d - c) / 7.0;

        Console.Write("y\\x  ");

        // Верхній рядок із значеннями x
        for (int i = 0; i < 8; i++)
            Console.Write($"{x + hx * i,8:F2}");

        Console.WriteLine();

        // Основна таблиця
        for (int i = 0; i < 8; i++)
        {
            Console.Write($"{y,5:F2}");
            x = a;

            for (int j = 0; j < 8; j++)
            {
                double u = Math.Sqrt(2 * x * y + 4 * y * y * y);
                Console.Write($"{u,8:F2}");
                x += hx;
            }

            Console.WriteLine();
            y += hy;
        }
    }
}
